### DISCUSSION TITLE
How to solve the problem of incorrect data directory correspondence after Minio restart

### DISCUSSION THREAD

**Discussion by smallemployees** [minio/minio#20203] [Q&A] [answered]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

## Expected Behavior

Is there any way to solve this problem

## Current Behavior

Minio was deployed using k8s, but after restarting the pod, drift occurred, resulting in an error in the data directory where Minio was suspended

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used (`minio --version`):
* Server setup and configuration:
* Operating System and version (`uname -a`):

**Comment by harshavardhana** [comment, marked as answer]
No idea what you are asking. If you are looking for pod affinity then you want to use local PVs or directpv without that using hostpath won't give you affinity to nodes.
